docs/quick_tutorial/authentication.rst

```diff
@@ -101,6 +101,31 @@ Steps
 
 #. Click the "Logout" link.
 
+#. Lets test if only the logged one can see ``Logout`` message:
+
+   .. literalinclude:: authentication/tutorial/tests.py
+    :linenos:
+
+#. Add  ``pyramid[pyramid]`` as a test dependency in ``authentication/setup.py``:
+
+   .. literalinclude:: authentication/setup.py
+    :language: python
+    :emphasize-lines: 11
+    :linenos:
+
+#. Install testing dependency:
+
+   .. code-block:: bash
+
+       $VENV/bin/pip install -e ".[testing]"
+
```

Comment:
Please remove lines 109-115. I think my recent setup.py changes will handle this.

Also please check that my narrative documentation changes do not already handle line 120. Else change line 120 to `$VENV/bin/pip install -e ".[test]"`.

Reply:
The tests look good. Thank you for your work on these!

Also please sign CONTRIBUTORS.txt, if you have not done so.